src/views/Search.vue

```diff
@@ -147,8 +147,6 @@ export default defineComponent({
 
 <style scoped>
   #scan-button {
-    position: fixed;
-    bottom: 10px;
-    width: 100%;
+    display: contents;
```

Comment:
use better solution here with absolute or fixed positioning.

Reply:
Done sir
----
In chrome :
![scan-button-chrome](https://user-images.githubusercontent.com/82817616/151830003-13396a36-cbc7-4712-a97b-819918847f04.png)

----
In firefox :

![scan-button-firefox](https://user-images.githubusercontent.com/82817616/151830089-b45555f5-2fcf-4509-914e-7a3b4c023609.png)